src/unix/mediactrl_gstplayer.cpp

```diff
@@ -178,6 +178,16 @@ static void realize_callback(GtkWidget* widget, wxGStreamerMediaBackend* be)
     gst_player_video_overlay_video_renderer_set_window_handle(GST_PLAYER_VIDEO_OVERLAY_VIDEO_RENDERER(be->m_video_renderer),
                                 wxGtkGetIdFromWidget(widget)
                                 );
+    GdkWindow* window = gtk_widget_get_window(widget);
+    if (strcmp("GdkWaylandWindow", g_type_name(G_TYPE_FROM_INSTANCE(window))) == 0) {
+        // on gtk wayland we need to place the video
+        int x, y;
+        GtkAllocation allocation;
+        gdk_window_get_origin(window, &x, &y);
+        gtk_widget_get_allocation(widget, &allocation);
+        gst_player_video_overlay_video_renderer_set_render_rectangle(GST_PLAYER_VIDEO_OVERLAY_VIDEO_RENDERER(be->m_video_renderer),
```

Comment:
Here it would be better to wrap this line but there is also what seems like a syntax error with the extraneous comma, so it's not just a style comment.
```suggestion
    gst_player_video_overlay_video_renderer_set_render_rectangle(
        GST_PLAYER_VIDEO_OVERLAY_VIDEO_RENDERER(be->m_video_renderer)
    );
```

Reply:
There are other arguments on the far right of the next line :)
I agree that style wasn't great, I just kept it in sync with `gst_player_video_overlay_video_renderer_set_window_handle` above -- I'll wrap it as well like you've suggested along with the other fixes today.